<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>547</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Analysis View</string>
  </property>
  <property name="windowIcon">
   <iconset theme="tools-wizard"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_meta">
      <attribute name="icon">
       <iconset theme="documentinfo"/>
      </attribute>
      <attribute name="title">
       <string>Meta</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="MetaPanel" name="widget_meta" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_basins">
      <attribute name="icon">
       <iconset theme="view-list-tree"/>
      </attribute>
      <attribute name="title">
       <string>Basins</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="BasinsPanel" name="widget_basins" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_tables">
      <attribute name="icon">
       <iconset theme="office-chart-line-stacked"/>
      </attribute>
      <attribute name="title">
       <string>Tables</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="TablesPanel" name="widget_tables" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_log">
      <attribute name="icon">
       <iconset theme="view-calendar-list"/>
      </attribute>
      <attribute name="title">
       <string>Logs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="LogPanel" name="widget_log" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_slot">
      <attribute name="icon">
       <iconset theme="office-chart-ring"/>
      </attribute>
      <attribute name="title">
       <string>Dataset</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="SlotPanel" name="widget_slot" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_filter">
      <attribute name="icon">
       <iconset theme="view-filter"/>
      </attribute>
      <attribute name="title">
       <string>Filter</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="FilterPanel" name="widget_filter" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_plot">
      <attribute name="icon">
       <iconset theme="draw-watercolor"/>
      </attribute>
      <attribute name="title">
       <string>Plot</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="PlotPanel" name="widget_plot" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterPanel</class>
   <extends>QWidget</extends>
   <header>shapeout2.gui.analysis</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MetaPanel</class>
   <extends>QWidget</extends>
   <header>shapeout2.gui.analysis</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotPanel</class>
   <extends>QWidget</extends>
   <header>shapeout2.gui.analysis</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlotPanel</class>
   <extends>QWidget</extends>
   <header>shapeout2.gui.analysis</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogPanel</class>
   <extends>QWidget</extends>
   <header>shapeout2.gui.analysis</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TablesPanel</class>
   <extends>QWidget</extends>
   <header>shapeout2.gui.analysis</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BasinsPanel</class>
   <extends>QWidget</extends>
   <header>shapeout2.gui.analysis</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
